src/splitpane/SplitPane.ts

```diff
@@ -0,0 +1,190 @@
+import { createHandle } from '@dojo/core/lang';
+import { debounce } from '@dojo/core/util';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { observeViewport } from '../common/util';
+import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
+import { v } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+
+import * as css from './styles/splitPane.m.css';
+
+/**
+ * Direction of this SplitPane
+ */
+export const enum Direction {
+	column,
+	row
+};
```

Comment:
None of our existing enum values are capitalized. If we want to standardize on this, we should probably do it in a separate PR.